### DISCUSSION TITLE
Is there any way to set a maximum for how far a draggable can go?

### DISCUSSION THREAD

**Discussion by growthwp** [mzabriskie/react-draggable#572] [Q&A]
I currently have a `Draggable` go on the `x` axis, but it just keeps and keeps on going. Is there no way to pass some props to make it stop at some point? I don't:

- Want it to go outside the parent.
- Want it to go past a certain threshold.

Any ideas?

**Comment by manten120** [comment]
The bounds property can be used to set the draggable range.

```js
<Draggable bouds={{left: 10, top: 20, right: 30, bottom: 40}}>
  {children}
</Draggable>
```

You can prohibit dragging outside of the div element.

```js
<div>
  <Draggable bouds="parent">
    {children}
  </Draggable>
<div>
```

**Reply by tornadicshark** [reply]
If you're trying to use this solution, please not it is "bounds", not "bouds"
